pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -417,35 +425,66 @@ func getInstallerPodName(revision int32, nodeName string) string {
 	return fmt.Sprintf("installer-%d-%s", revision, nodeName)
 }
 
-// ensureInstallerPod creates the installer pod with the secrets required to if it does not exist already
-func (c *InstallerController) ensureInstallerPod(nodeName string, operatorSpec *operatorv1.OperatorSpec, revision int32) error {
-	required := resourceread.ReadPodV1OrDie([]byte(installerPod))
-	required.Name = getInstallerPodName(revision, nodeName)
-	required.Namespace = c.targetNamespace
-	required.Spec.NodeName = nodeName
-	required.Spec.Containers[0].Image = c.installerPodImageFn()
-	required.Spec.Containers[0].Command = c.command
-	required.Spec.Containers[0].Args = append(required.Spec.Containers[0].Args,
-		fmt.Sprintf("-v=%d", 4),
+// arrToYAML converts a string array to YAML array
+// Example: []string{"1", "2"} will convert to: '["1","2"]'
+func arrToYAML(arr []string) string {
```

Comment:
probably should be some string helper somewhere?

Reply:
json.Marshal would do, no?

Reply:
good idea